### PR TITLE
send user,se,customerType headers on call to Ruby api for Express.js and Spring-boot

### PR DESCRIPTION
**Pull request by gid-sentry**
Adding some headers to be sent to the ruby API when requests go through from the spring-boot backend.

### CODE REVIEW

File: express/server.js [hunk 1]
```diff
@@ -37,22 +37,26 @@ const DB = require('./db');
...
   const customerType = req.headers.customertype;
   if(!['undefined'].includes(customerType)) {
     Sentry.setTag("customerType", customerType);
+    headers["customerType"]=customerType
   }
 
   const email = req.headers.email;
   if(!['undefined'].includes(email)) {
     Sentry.setUser({ 'email': email })
+    headers["user"]=email
```
Comment (thinkocapo) [inline express/server.js:57]: Could you make it `headers["email"]` instead? As this is what ruby [is looking for](https://github.com/sentry-demos/application-monitoring/blob/master/ruby/main.rb#L34). 'email' is always what the React app [sets it as](https://github.com/sentry-demos/application-monitoring/blob/master/react/src/components/Products.js#L35). If there is code on the frontend setting it as 'user' then please let me know and I'll have a look.

Comment (thinkocapo) [reply express/server.js:57]: @gid-sentry

Comment (gid-sentry) [reply express/server.js]: Done in [13c1ca8](https://github.com/sentry-demos/application-monitoring/pull/86/commits/13c1ca8dd10d5b017588278df804b170089ff807)

**Comment by gid-sentry** [comment]
Transaction example for express.js:
https://sentry.io/organizations/testorg-az/discover/apm-express:90411c59a60c4f21ab0d23e2d6b0fc1c/?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&query=&sort=-timestamp&statsPeriod=24h&yAxis=count%28%29

Transaction example for Springboot:
https://sentry.io/organizations/testorg-az/performance/apm-springboot:8e221dc21c6b45cbbc07853e6d2d49cf/?project=6609503&query=transaction.duration%3A%3C15m&statsPeriod=7d&transaction=GET+%2Fapi&unselectedSeries=p100%28%29

**Comment by gid-sentry** [comment]
Springboot transaction example with SE set:
https://sentry.io/organizations/testorg-az/discover/apm-springboot:44ad880a1ad1473dbd420cda67a123a0/?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&project=6609503&query=&sort=-timestamp&statsPeriod=24h&yAxis=count%28%29

**Review by thinkocapo — APPROVED** [review]

**Pull request merged**